thirdeye-ui/src/app/utils/routes/routes.util.ts

```diff
@@ -23,7 +23,7 @@ const PLACEHOLDER_ROUTE_ID = ":id";
 
 export const AppRouteRelative = {
     ADMIN: "admin",
-    COHORT_DETECTOR: "cohort-detector",
+    COHORT_DETECTOR: "cohort-recommender",
```

Comment:
Rename this from `COHORT_DETECTOR` => `COHORT_RECOMMENDER` and ensure all usages of this is refactored as well

Reply:
Updated.